lib/oban/crontab/scheduler.ex

```diff
@@ -51,6 +51,7 @@ defmodule Oban.Crontab.Scheduler do
 
   @impl GenServer
   def handle_continue(:start, state) do
+    enqueue_reboot(state.conf)
```

Comment:
Let's call these `insert` instead of `enqueue`, for consistency with the query it's calling.

```suggestion
    insert_reboot(state.conf)
```